Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -0,0 +1,102 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Globalization;
+using System.Linq;
+using Windows.ApplicationModel;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.System;
+using Windows.System.Profile;
+using Windows.System.UserProfile;
+
+namespace Microsoft.Toolkit.Uwp.Helpers
+{
+    /// <summary>
+    /// Defines class providing information of OS and application
+    /// </summary>
+    public static class SystemInformation
+    {
+        /// <summary>
+        /// Gets Application's name
+        /// </summary>
+        public static string ApplicationName { get; }
+
+        /// <summary>
+        /// Gets Application's version
+        /// </summary>
+        public static PackageVersion ApplicationVersion { get;  }
+
+        /// <summary>
+        /// Gets the most preferred culture by the user
+        /// </summary>
+        public static CultureInfo Culture { get; }
```

Comment:
@deltakosh 'Never too cautious' is good, but I like to avoid putting try-catch everywhere. In this case I woun't be afraid of null, hence there is a check-up for language presence, but CultureNotFound exception can be a worse one in this case. I've put the block, just in case.